### PR TITLE
style(descriptionlist): typography styles were susceptible to being i…

### PR DESCRIPTION
**Pull request by fortunato**
…ncorrectly inherited

AIP-2092

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
 # Changelog
```
Comment (vdagraca) [inline CHANGELOG.md:2]: Is this automatically generated?

File: src/common/src/DescriptionList/DescriptionList.test.tsx [hunk 2]
```diff
@@ -41,4 +41,9 @@ describe('<DescriptionList />', () => {
 		const { getByText } = render(<DescriptionList list={[...list, { label: '', value: <DummyReactComponent /> }]} />);
 		expect(getByText('__REACT_COMPONENT__')).toBeInTheDocument();
 	});
+
+	it('optionally renders a footer with a React node', () => {
+		render(<DescriptionList list={[...list, { label: '', value: <DummyReactComponent /> }]} footer={<>[FOOTER]</>} />);
+		expect(screen.getByTestId('description-footer').textContent).toEqual('[FOOTER]')
```
Comment (vdagraca) [inline src/common/src/DescriptionList/DescriptionList.test.tsx:47]: 'screen' was new for me, but looked it up, nice short solution!

**Review by vdagraca — APPROVED** [review]

File: src/common/src/DescriptionList/DescriptionList.test.tsx (hunk 2 above)
Comment (fortunato) [reply src/common/src/DescriptionList/DescriptionList.test.tsx:47]: Yeah, makes it a little easier; no more preparing/keeping a reference in scope :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (fortunato) [reply CHANGELOG.md:2]: It certainly is (by standard-version) as long as we stick to the commit message convention: https://www.conventionalcommits.org/en/v1.0.0/
`npm run commit` is ur friend :slightly_smiling_face:

**Pull request merged**